package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
-    "build": "rimraf ./dist && mkdir dist && tsc --outDir dist && git rev-parse HEAD > BUILD_SHA",
```

Comment:
Any reason to drop the BUILD_SHA file? I found it helpful for debugging

Reply:
I thiught it is an artifact. What is the purpose of this? Do you use it in production to determine the codebase by git commit hash?

Reply:
Yeah! I was having some trouble with the build pipeline at one point where built assets weren't being committed, so it was nice to validate what SHA had actually been built. Its not strictly necessary, but it was handy at that point and I feel like doesn't really cost us anything to keep